src/modules/iotjs_module_process.c

```diff
@@ -464,6 +475,30 @@ JS_FUNCTION(ForceGC) {
   return jerry_create_boolean(true);
 }
 
+JS_FUNCTION(StartUVCheck) {
+  int status;
+  if (check_handle.data != NULL) {
+    status = uv_check_start(&check_handle, onUVCheckCallback);
```

Comment:
Suggestion function name: `UVCheckCallback`, the `on` is duplicated with `callback`.